src/Runner/PhpInterpreter.php

```diff
@@ -12,6 +12,11 @@
 {
 
 	/**
+	 * @param  string
+	 */
+	function addArguments($args);
```

Comment:
I think that it would better to remove this from the interface and make it `protected` on `AbstractInterpteter`.